src/plugins/vis_augmenter/common/types.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ExpressionFunctionDefinition } from '../../expressions';
+
+export interface VisLayer {
+  // will be used as the column ID
+  id: string;
+  // will be used as the name when hovering over the tooltip
+  name: string;
+}
+
+export type VisLayers = VisLayer[];
```

Comment:
nit - not sure it's more readable than the definition

Reply:
I had only added this to make a cleaner interface when defining the expression interface: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/4ff4451a2b8397d2325696733763117762a93c78/src/plugins/expressions/common/expression_types/specs/vis_layers.ts#L13